docs/guides/service_configuration_file.md

```diff
@@ -1,10 +1,8 @@
-The service configuration file `service.yaml` is a YAML file where the main attributes of the agent service are set up, including the particular agent that composes the service. An important feature of service configuration files is that they can override attributes defined in agent or component configurations.
+!!! warning
 
+    This is advanced documentation. You can skip this section if it's the first time you are following the guides.
 
-<figure markdown>
-![](../images/development_process_define_service.svg)
-<figcaption>Part of the development process covered in this guide</figcaption>
-</figure>
+The service configuration file `service.yaml` is a YAML file where the main attributes of the agent service are set up, including the particular agent that composes the service. An important feature of service configuration files is that they can override attributes defined in agent or component configurations.
```

Comment:
"the particular agent that composes the service" -> "the particular agent that defines the service"

Reply:
"if a component uses another component, the former can override configuration values of the latter." -> "if a component uses another component as a sub-component, the former can override configuration values of the sub-component."

In the image **Overrides precedence** replace "Other component" by "Sub-component"